pages/guides/sandbox/index.mdx

```diff
@@ -151,7 +151,7 @@ To help you test error scenarios, a flag and an option can be used in the first
 
 #### Test micro-deposit verification
 
-If your application leverages the micro-deposit method of bank verification, Dwolla will transfer two deposits of less than \$`0.10` to your customer's linked bank or credit union account after calling the API to initiate micro-deposits. Since the Sandbox environment doesn't replicate any bank transfer processes, any two amounts **below** \$`0.10` will allow you to verify the funding source immediately.
+If your application leverages the micro-deposit method of bank verification, Dwolla will transfer two deposits of less than `$0.10` to your customer's linked bank or credit union account after calling the API to initiate micro-deposits. Since the Sandbox environment doesn't replicate any bank transfer processes, any two amounts **below** `$0.10` will allow you to verify the funding source immediately.
```

Comment:
Placing`$` within the inline code because the `\$` was being rendered literally.